src/syslog_mcp/tools/interface/security_tools.py

```
@@ -0,0 +1,162 @@
+    format_suspicious_activity_summary,
+    format_auth_timeline_summary,
+    format_ip_reputation_summary
+)
+from ...utils.logging import get_logger
+
+logger = get_logger(__name__)
+
+
+async def get_failed_auth_summary(
+    client,
+    device: Optional[str] = None,
```

Ruff (0.12.2)

Use `X | None` for type annotations

Convert to `X | None`

(UP045)